src/Test.Utilities.Sarif/FileDiffingUnitTests.cs

```diff
@@ -228,15 +229,27 @@ protected virtual string ConstructTestOutputFromInputResource(string inputResour
 
             // Reify the list of keys because we're going to modify the dictionary in place.
             List<string> keys = expectedSarifTextDictionary.Keys.ToList();
+            var userFacingTexts = new Dictionary<string, string>();
 
             foreach (string key in keys)
             {
+                string actualSarifText = actualSarifTextDictionary[key];
+                SarifLog actualSarifLog = JsonConvert.DeserializeObject<SarifLog>(actualSarifText);
+                if (actualSarifLog.Runs?[0].TryGetProperty("consoleOut", out string userFacingText) == true)
```

Comment:
>TryGetProperty [](http://example.com/codeflow?start=44&length=14)

So here's the trick upstream users of this base class can do. If they'd like, they can populate the run with `consoleOut` to capture user-facing strings. We can (and should) build this deeper into the test infra but it will break everyone's tests (which will require baselining). So we're taking this spotfix for now.